spec/models/task_spec.rb

```diff
@@ -1,5 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
+    it 'is valid with all attributes' do
+      user = FactoryBot.create(:user)
+      task = FactoryBot.build(:task, user: user)
+      expect(task).to be_valid
+      expect(task.errors).to be_empty
+    end
+
+    it 'is invalid without title' do
+      user = FactoryBot.create(:user)
+      task = FactoryBot.build(:task, title: "",  user: user)
```

Comment:
設定ファイルに追記を行うことで、`FactoryBot.`の部分は省略することができます！
```suggestion
      task = build(:task, title: "",  user: user)
```

Reply:
fe2c982